synapse/handlers/message.py

```diff
@@ -1474,21 +1474,23 @@ async def handle_new_client_event(
 
         # We now persist the event (and update the cache in parallel, since we
         # don't want to block on it).
-        event, context = events_and_context[0]
+        # Note: mypy gets confused if we inline dl and check with twisted#11770.
+        # Some kind of bug in mypy's deduction?
+        dl = (
```

Comment:
I don't get what `dl` stands for, can we use a longer variable name?

Reply:
I think it is "deferred list"?

Reply:
Yes, deferred list.

Thanks for getting this over the line.